client/src/components/layouts/prism.css

```
@@ -7,6 +7,11 @@ pre[class*='language-'] {
   background: var(--primary-background);
 }
 
+pre[class*='language'] code[class*='language-'] {
+  overflow-wrap: anywhere;
+  white-space: pre-wrap;
+}
+
```

```suggestion
pre[class*='language'] code[class*='language-'] {
  white-space: nowrap;
  overflow: auto;
}

.video-quiz-option {
  max-width: 90%;
}
```

I spoke with @ahmadabdolsaheb and it does sound like horizontal scroll is the desired approach.

This CSS change will preserve the horizontal scroll that exists on the code within a video challenge question, while adding it to the video challenge answer. It also shrinks the answer text width a bit to account for the select button.

Certainly open to better CSS approaches for this same effect, though 🙂